Dockerfile

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright 2019-present Open Networking Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG BASE_OS=ubuntu:16.04
+
+FROM $BASE_OS AS builder
+WORKDIR /openmme
+COPY install_builddeps.sh .
+RUN ./install_builddeps.sh
+
+COPY . ./
+ENV LOG_LEVEL=LOG_DEBUG
```

Comment:
Should this be present in the all versions? should we control it from outside and publish a debug tag?
@hyunsun @zdw

Reply:
I assume this comment is about the logging level - which depends on the logging method used.

In other projects the logging config is more complex (for example, logging in JSON to a kafka topic) so the configuration is passed to the component as a file mounted in the container.

Reply:
I agree that logging level should be controlled from outside or even better if it could be passed as a runtime parameter. Currently it is hard coded to WARN level, so I made another PR to add ability to configure it as a build parameter https://github.com/omec-project/openmme/pull/18. I'll remove the log level DEBUG here and hope to keep discuss about logging level in the other PR.